### PR TITLE
PDA Games Update

### PR DESCRIPTION
**Pull request by DeityLink**
Adds 3 new purchasable apps on PDA terminals:

-Snake II
snake

-Minesweeper
feels_like_1995 - copie

-Spess Pets
pet

### CODE REVIEW

**Comment by 9600bauds** [comment]
yes

File: code/game/gamemodes/gameticker.dm [hunk 1]
```diff
@@ -424,6 +429,32 @@ var/global/datum/controller/gameticker/ticker
 			return F
 
 
+/datum/controller/gameticker/proc/init_PDAgames_leaderboard()
+	init_snake_leaderboard()
+	init_minesweeper_leaderboard()
+
+/datum/controller/gameticker/proc/init_snake_leaderboard()
+	for(var/x=1;x<=9;x++)
```
Comment (ComicIronic) [inline code/game/gamemodes/gameticker.dm]: So you have 9 slots on the leaderboard, and then 8 slots on each slot?
Why?

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 2]
```diff
@@ -1028,6 +1028,292 @@ var/global/list/obj/item/device/pda/PDAs = list()
 							dat += {"<li>[mkr.name] ([mkr.x]/[mkr.y]) <a href='byond://?src=\ref[src];choice=removeMarker;rMark=[mkr.num]'>remove</a></li>"}
 						dat += {"</ul>"}
 
+			if (105)//Snake II
```
Comment (ComicIronic) [inline code/game/objects/items/devices/PDA/PDA.dm]: Why aren't these defines?

File: code/game/gamemodes/gameticker.dm (hunk 1 above)
Comment (DeityLink) [inline code/game/gamemodes/gameticker.dm]: 9 speeds, 8 labyrinths

Comment (ComicIronic) [inline code/game/gamemodes/gameticker.dm]: Ah. I suggest you make them defines, then.

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 2 above)
Comment (DeityLink) [inline code/game/objects/items/devices/PDA/PDA.dm]: Do you know how many modes there are on the PDA? I'm not making defines for every single of them when I'm only adding 3 new modes. Furthermore, I made apps modes as simple as I could (101,102,103, etc) unlike those cartridges modes. Curses to whoever coded the PDA cartridges.

File: code/game/gamemodes/gameticker.dm (hunk 1 above)
Comment (DeityLink) [inline code/game/gamemodes/gameticker.dm]: Why? there one ever be more or less speeds/labyrinths. Isn't the main point of defines being that you can easily change values?

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 2 above)
Comment (ComicIronic) [inline code/game/objects/items/devices/PDA/PDA.dm]: Then at least make the apps you're adding defines.

I'm honestly disappointed the first apps you added are magic-number'd as well.

File: code/game/gamemodes/gameticker.dm (hunk 1 above)
Comment (ComicIronic) [inline code/game/gamemodes/gameticker.dm]: And also for readability, so people know WHY a value is a value, rather than it just being so.

**Comment by CptWad** [comment]
Make sure the captain gets these at round start.

**Comment by DeityLink** [comment]
The captain automatically gets all the PDA apps installed on his PDA at round start. That's how I've been testing them (since otherwise you need a money account to purchase them and creating one is a pain when you have to do it 500 times)

**Comment by DeityLink** [comment]
welp, we're good to go.

**Comment by clusterfack** [comment]
Ahhh 170 files how

**Comment by ComicIronic** [comment]
You have a copy of a file commited.

**Comment by DeityLink** [comment]
most of the files are PNGs that's how

got rid of the duplicate file

**Comment by DeityLink** [comment]
oh boy oh boy oh boy oh boy

**Comment by Iamgoofball** [comment]
MINESWEEPER

MINESWEEEEEEEEPPPPPPEEEEEEERRRRRRRRRRR

:+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1: :+1:

**Comment by Man-Shroom** [comment]
Oh yeah :+1:

**Pull request merged**